docs/build-tools/sbt.md

```diff
@@ -49,11 +49,11 @@ addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "@BLOOP_VERSION@")
 Next, run:
 
 ```
-sbt "set bloopExportJarClassifiers := Some(Set(\"source\"))" bloopInstall
+sbt -Dbloop.export-jar-classifiers="sources" bloopInstall
 ```
 
-to generate the Bloop JSON configuration files. You can also set the 
-`bloopExportJarClassifiers` setting inside your main build.sbt file, but using
+to generate the Bloop JSON configuration files. You can also set the property
+`bloop.export-jar-classifiers` inside your main build.sbt file, but using
```

Comment:
Let's keep it as is. It's better to add the setting inside of the `build.sbt` than each time when running